velox/docs/monthly-updates/february-2023.rst

```diff
@@ -0,0 +1,77 @@
+********************
+February 2023 Update
+********************
+
+
+Core Library
+============
+
+ * Add several memory related fixes: task reference in partition output iobuf to prevent memory leaking,
+   memory corruption caused by ntile.
+ * Add concurrent memory allocation benchmark.
+ * Add various enhancements such as replace ArrayBuilder class with ArrayWriter, support for decimal types in DecodedVector,
+  make ConstantTypedExpr safer to use, Require type in FlatVector constructor, support to serialize and deserialize long decimals,
+  BloomFilter to support serialization and memory tracking
+ * Add fixes for null handling of decimal values in QueryAssertions, Decimal Sum Aggregate,
+ * Remove usage of variant::inferType from variantArrayToVector function, variant::inferType from core::ConstantExpr, BaseVector::isConstant(rows) method
+ * Add fixes for expression compilation and common sub-expression evaluation
+ * Add ArrowStream operator and support date type in Arrow bridge
```

Comment:
New functionality goes first, fixes go next. Make each item full sentence, i.e. period at the end.

* Add ArrowStream operator.  (add a link to documentation)
* Add support for DATE type in Arrow bridge. (move this to separate Arrow section next to Substrait)
* Add support for k PRECEDING/FOLLOWING window frames in ROWS mode.